CMakeLists.txt

```diff
@@ -536,6 +536,9 @@ if(WIN32)
 endif(WIN32)
 
 # Extern
+set(RECASTNAVIGATION_DEMO OFF CACHE BOOL "Do not build RecastDemo")
+
+add_subdirectory (extern/recastnavigation)
```

Comment:
Can you also set to building statically by default? We should be pulling this into component.

Reply:
Fixed